content/en/tracing/trace_collection/library_config/_index.md

```diff
@@ -3,14 +3,117 @@ title: Configure the Datadog Tracing Library
 type: multi-code-lang
 ---
 
-For information about configuration options for tracing libraries, choose your language:
+This page describes configuration options that are supported across all SDKs. To view these common configuration options, see [Common configuration options](#common-configuration-options).
+
+For configuration options specific to your programming language, choose your SDK from the options below:
 
 {{< partial name="apm/apm-compatibility.html" >}}
 
 <br>
 
 To instrument an application written in a language that does not yet have official library support, see the list of [community tracing libraries][1].
 
+## Common configuration options
+The following configuration options behave consistently across multiple Datadog SDKs:
+
+### Traces
+
+`DD_TRACE_<INTEGRATION>_ENABLED`
+: **Default**: `true` <br>
+**Supported Input**: Boolean <br>
+**Not Supported In**: Go SDK<br>
+**Description**: Enables or disables instrumentation for the specified `<INTEGRATION>`. The integration name must be in uppercase (for example, `DD_TRACE_KAFKA_ENABLED=true`)
+
+`DD_TRACE_RATE_LIMIT`
+: **Default**: `100` <br>
+**Supported Input**: A positive integer<br>
+**Description**: Sets the maximum number of traces to sample per second; applies only when either `DD_TRACE_SAMPLING_RULES` or `DD_TRACE_SAMPLE_RATE` is set.
+
+`DD_TRACE_HEADER_TAGS`
```

Comment:
Can we provide examples here? From reading, the exact value I'd want to set here is unclear

Reply:
@bm1549 are the examples under Supported Input and Description below not clear enough?

If not, what kind of examples are you looking for?

Reply:
The example is great. Thanks for adding!